packages/host/app/components/pill-menu/index.gts

```diff
@@ -168,12 +170,69 @@ export default class PillMenu extends Component<Signature> {
         overflow-y: auto;
         min-height: 0;
       }
+
+      .menu-content::before,
+      .menu-content::after,
+      .menu-footer::before {
+        content: '';
+        display: block;
+        width: 100%;
+        height: 5px;
+        position: absolute;
+        left: 0;
+        opacity: 0;
+        pointer-events: none;
+      }
+
+      .menu-content::before {
+        background: linear-gradient(
+          to bottom,
+          rgba(0, 0, 0, 0.25) 0%,
+          transparent 100%
+        );
+
+        animation: scroll-pill-menu-content linear forwards;
+        animation-timeline: --pill-menu-content-scroll-timeline;
+      }
+
+      .pill-menu.has-footer .menu-content::after {
+        display: none;
+      }
+
+      .menu-content::after {
+        background: linear-gradient(
+          to top,
+          rgba(0, 0, 0, 0.25) 0%,
+          transparent 100%
+        );
+
+        animation: scroll-pill-menu-content reverse linear backwards;
+        animation-timeline: --pill-menu-content-scroll-timeline;
+
+        bottom: var(--boxel-sp-sm);
+        margin-bottom: 16px;
```

Comment:
This and the `bottom: 60px` are more hackish than I’m comfortable, there’s something about the element structure of these components I don’t understand that makes positioning difficult:

![screencast 2025-07-09 11-43-49](https://github.com/user-attachments/assets/4d0fb231-24bd-4291-a25d-d471d43c7bc8)

The header and footer elements don’t have padding themselves, the container does, so the gradients have to be positioned with offsets. I plan to improve this in a followup PR.